src/hostapi/pulseaudio/pa_linux_pulseaudio.c

```diff
@@ -952,6 +952,22 @@ PaError OpenStream( struct PaUtilHostApiRepresentation *hostApi,
         goto error;
     }
 
+    /* Allocate memory for source and sink names. */
+    char defaultSourceStreamName[] = "Portaudio source";
```

Comment:
I think these should be const char * as they are allocated and they are not changed anymore

Reply:
Updated to use const.